### PR TITLE
[Docs][Connector-V2][PostgreSQL] Refactor connector-v2 docs using unified format PostgreSQL #4590

### PR DESCRIPTION
**Pull request by zhilinli123**
## Purpose of this pull request
subtask of https://github.com/apache/incubator-seatunnel/issues/4572

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/PostgreSql.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+```
+# Defining the runtime environment
+env {
+  # You can set flink configuration here
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
```
Comment (Hisoka-X) [inline docs/en/connector-v2/sink/PostgreSql.md]: Please remove comment

File: docs/en/connector-v2/source/PostgreSQL.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        url = "jdbc:postgresql://localhost:5432/test"
+        driver = "org.postgresql.Driver"
+        user = "root"
+        password = "test"
+        query = "select * from source limit 16"
+    }
+}
+
+transform {
+    # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
```
Comment (Hisoka-X) [inline docs/en/connector-v2/source/PostgreSQL.md]: same as above

Comment (zhilinli123) [reply docs/en/connector-v2/source/PostgreSQL.md]: Done

**Comment by zhilinli123** [comment]
@Hisoka-X  PTAL

File: docs/en/connector-v2/sink/PostgreSql.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        password = 123456
+        query = "insert into test_table(name,age) values(?,?)"
+    
+        is_exactly_once = "true"
+    
+        xa_data_source_class_name = "org.postgresql.xa.PGXADataSource"
+    }
+}
+```
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/PostgreSql.md:179]: Can you add  examples for `generate_sink_sql `

Comment (zhilinli123) [reply docs/en/connector-v2/sink/PostgreSql.md:179]: Line 149 has been added @EricJoy2048

File: docs/en/connector-v2/sink/PostgreSql.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+        driver = "org.postgresql.Driver"
+        user = root
+        password = 123456
+        query = "insert into test_table(name,age) values(?,?)"
+     }
+  # If you would like to get more information about how to configure seatunnel and see full list of sink plugins,
+  # please go to https://seatunnel.apache.org/docs/category/sink-v2
+}
+```
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/PostgreSql.md:138]:
```suggestion

### Generate Sink SQL
```

Comment (zhilinli123) [reply docs/en/connector-v2/sink/PostgreSql.md:138]: Done

**Review by EricJoy2048 — DISMISSED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Review by Carl-Zhou-CN — APPROVED** [review]
+1

**Review by ic4y — APPROVED** [review]

**Pull request merged**